iOS-training/ContentView.swift

```diff
@@ -97,12 +104,15 @@ struct ContentView: View {
     }
 
     func fetchWeather() {
```

Comment:
[imo] `.task`は非同期関数を扱えるため、
```suggestion
    func fetchWeather() async {
```
の様にする事もできます